src/components/BrowseJobInDetails/BrowseJobInDetails.jsx

```diff
@@ -1,14 +1,51 @@
+/* eslint-disable */
 import React from 'react';
 import PropTypes from 'prop-types';
 import JobDetailsCard from './JobDetailsCard';
 import {ClosedChatBox, OpenedChatBox} from '../../components';
 import {downarrow} from '../../components/Assets';
-import {Link} from 'react-router-dom';
+import {Link, useLocation} from 'react-router-dom';
+
+const BrowseJobInDetails = ({isFreelancer, isOpen, setIsOpen, jobData}) => {
+  const location = useLocation();
+
+  console.log("jobData in BrowseJobsInDetails:", jobData);
+ 
+
+  const handleManagePostedJobsClick = () => {
+    console.log("des", description)
+    const jobId = localStorage.getItem('job_id');
+    const accessToken = localStorage.getItem('access_token');
+
+    const apiUrl = 'http://35.154.4.80:3000/api/v0/users/me/jobs?page=1&per_page=8'; 
```

Comment:
The URL is hardcoded, let's change that. Also the pagination is hardcoded as well. We have to integrate the logic for pagination.

Reply:
When we scroll past a point in the page, the next page data needs to be requested from the backend.